### ISSUE TITLE
Required libraries are not listed in requirements.txt

### ISSUE THREAD

**Issue by aurel-g** [smthemex/ComfyUI_Diffree#7]
These libraries are needed to load this custom node pack but are not listed in requirements.txt (nor in comfyui's requirements) :

- omegaconf
- scikit-image
- opencv-python

Example of output when launching ComfyUI :

```
Launching ComfyUI from: /mnt/comfyui/app

Total VRAM 16073 MB, total RAM 32000 MB
pytorch version: 2.5.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4060 Ti : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: /mnt/comfyui/app/web
Traceback (most recent call last):
  File "/mnt/comfyui/app/nodes.py", line 2035, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/mnt/comfyui/app/custom_nodes/ComfyUI_Diffree/__init__.py", line 2, in <module>
    from .Diffree_node import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "/mnt/comfyui/app/custom_nodes/ComfyUI_Diffree/Diffree_node.py", line 6, in <module>
    from omegaconf import OmegaConf
ModuleNotFoundError: No module named 'omegaconf'

Cannot import /mnt/comfyui/app/custom_nodes/ComfyUI_Diffree module for custom nodes: No module named 'omegaconf'

Import times for custom nodes:
   0.0 seconds: /mnt/comfyui/app/custom_nodes/websocket_image_save.py
   0.0 seconds (IMPORT FAILED): /mnt/comfyui/app/custom_nodes/ComfyUI_Diffree

Starting server

To see the GUI go to: http://0.0.0.0:8080
```

**Comment by smthemex** [comment]
yeah ，i will fix it

**Comment by aurel-g** [comment]
Fixed by PR #8, thanks
